.circleci/config.yml

```diff
@@ -326,6 +327,23 @@ jobs:
           working_directory: ~/expo
           command: fastlane android prod_release
 
+  docs:
+    executor: nix
+    resource_class: xlarge # TODO: Use less than 8Gib of ram to build a static site
+    steps:
+      - setup
+      - update_submodules
+      - yarn_install:
+          working_directory: docs
+      - yarn:
+          command: export
+          working_directory: docs
+      - deploy:
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              aws s3 sync docs/out s3://expo-static-docs-test --delete
```

Comment:
One thing I'm worried about is how safe deployments are. Namely, if a user is loading the docs site or has loaded the docs site, they may have HTML/JS files that reference other JS files. When the new site is deployed and we delete the old files from S3, HTTP requests for the old files may fail if they are not cached by CloudFront, causing browser errors.

I think we need to keep the old files around, at least for a period of time (a day is probably fine, though a week could be safe). I know that S3 has a way to autodelete files after a certain timeframe, but we'd like to keep the most recent version of the site around. Is there a way on S3 to say "everything I'm uploading is version n+1; please delete all files with older version numbers once they are a week old"?